scss/_utilities_text-align.scss

```diff
@@ -10,19 +10,22 @@
 @mixin vf-u-text-align--center {
   .u-text-align--center {
     text-align: center !important;
+    justify-content: center !important;
```

Comment:
Without setting `justify-content` the text will be left aligned also. I suggest the name of the utility is updated to `.u-content-align` as it is what its doing. CSS have named the attribute incorrectly as it does align any block elements.

Either way, this need to be applied to this utility or the renamed one as it keeps the behaviour consistent.